### PR TITLE
Jared/hooktesting

### PR DESCRIPTION
**Pull request by jared-2016**
# Description

Apologies in advance this is a big PR and I didn't see anyway around it without making verbose and redundant mini PR's. Push through the pain and get it all done in one blow.

run code formatter on all samples to avoid githooks constantly making little changes to peoples PR's which causes confusion. Also updated the templates so that the follow the new formatting so that the commit hooks work on the sample source and header files.

## Type of change

- [x] Bug fix?(git hooks were not working so i guess this applies)
- [ ] New sample implementation
- [ ] Sample viewer enhancement
- [x] Other enhancement

**Platforms tested on**:

- [ ] Windows
- [ ] Android
- [ ] Linux
- [x] macOS
- [ ] iOS

## Checklist

- [ ] Runs and compiles on all active platforms as a standalone sample
- [ ] Runs and compiles in the sample viewer(s)
- [ ] Branch is up to date with the latest main/v.next
- [ ] All merge conflicts have been resolved
- [ ] Self-review of changes
- [ ] There are no warnings related to changes
- [ ] No unrelated changes have been made to any other code or project files
- [ ] Code is commented with correct formatting (CTRL+i)
- [ ] All variable and method names are camel case
- [ ] There is no leftover commented code
- [ ] Screenshots are correct size and display in description tab (500px by 500px, platform agnostic)
- [ ] If adding a new sample, it is added to the sample viewer
- [ ] Cherry-picked to Main branch (if applicable)

### CODE REVIEW

**Comment by jared-2016** [comment]
Forgot to have the githook run on all the main.cpp files for each individual sample. That's currently IP locally for me.

**Comment by JamesMBallard** [comment]
Given the size of this PR, I recommend waiting until we finalize v.next for the 200.5.0 release. Our next samples build will likely be the last one for 200.5.

File: CppSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.cpp [hunk 1]
```diff
@@ -18,22 +18,25 @@
...
 #endif // PCH_BUILD
 
+// sample headers
 #include "AnalyzeHotspots.h"
 
-#include "Map.h"
-#include "MapQuickView.h"
-#include "MapTypes.h"
-#include "GeoprocessingTask.h"
+// C++ API headers
```
Comment (JamesMBallard) [inline CppSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.cpp:24]: This is picky, but "C++ API" is how we internally differentiate between our internal API and our public-facing API. It's a fairly generic phrase for samples, so I would recommend using "ArcGIS Maps SDK headers" or something like that in the samples repo only.

**Review by JamesMBallard — COMMENTED** [review]
I did a quick scan, but I think we need to tweak the wording of one of the include sections. It's too generic for samples.

File: CppSamples/Analysis/AnalyzeHotspots/main.cpp [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
 // limitations under the License.
+// [Legal]
 
-#include <QGuiApplication>
-#include <QQuickView>
-#include <QCommandLineParser>
+// sample headers
+#include "AnalyzeHotspots.h"
+
+// ArcGIS Maps SDK headers
```
Comment (JamesMBallard) [inline CppSamples/Analysis/AnalyzeHotspots/main.cpp:19]: Much better name, thanks for making the change.

**Review by JamesMBallard — APPROVED** [review]

**Pull request merged**